app/templates/application.hbs

```diff
@@ -12,6 +12,17 @@
 <SideBar>
   <div class="footer-pusher">
     <NavBar @unreadNotifications={{this.unreadNotifications}} />
+    
+     <div class="flash-messages ui container">
+      {{#each flashMessages.queue as |flash|}}
+         <div class="ui negative message">
```

Comment:
Flash message can be of any type. Not just danger/negative. Modify UI accordingly

Reply:
Ok